crates/ibc/src/core/ics04_channel/error.rs

```diff
@@ -351,6 +351,10 @@ define_error! {
         AbciConversionFailed
             { abci_event: String }
             | e | { format_args!("Failed to convert abci event to IbcEvent: {}", e.abci_event)},
+
+        Other
```

Comment:
We already have the [`AppModule` error](https://github.com/cosmos/ibc-rs/blob/0c5eb875f77f0de3360e58db957e8814aabbee60/crates/ibc/src/core/ics04_channel/error.rs#L343) for IBC applications.

What is the intended use of this error?

Reply:
Note: We also have the [`ImplementationSpecific` error](https://github.com/cosmos/ibc-rs/blob/249650d0c396f16e6bdc43ae9e5fb9892580082f/crates/ibc/src/core/ics02_client/error.rs#L49) for clients, which I believe serves the same purpose as `Other` here... But I prefer `Other` as a name actually!

Reply:
> We already have the [`AppModule` error](https://github.com/cosmos/ibc-rs/blob/0c5eb875f77f0de3360e58db957e8814aabbee60/crates/ibc/src/core/ics04_channel/error.rs#L343) for IBC applications.
> 
> What is the intended use of this error?

yet, for Application impl is ok, but for ics02, 03, 04 interface to bad